documentation/source/Device.md

```diff
@@ -0,0 +1,179 @@
+# Training modes
+
+SuperGradients allows users to train models on different modes:
+- CPU
+- single GPU - (CUDA)
+- single GPU - Data Parallel (DP)
+- multiple GPUs' - Distributed Data Parallel (DDP)
+
```

Comment:
Please give some intro on DP and DDP followed by torch links for more details.

Reply:
DP:
https://github.com/Deci-AI/super-gradients/blob/7e6af4cc70e0b27c7070f7e3f534a07aaa996dc6/documentation/source/Device.md?plain=1#L35-L54

DDP:
https://github.com/Deci-AI/super-gradients/blob/7e6af4cc70e0b27c7070f7e3f534a07aaa996dc6/documentation/source/Device.md?plain=1#L74-L87